index.html

```diff
@@ -353,6 +353,60 @@
   <section id="conformance">
     <p>There is only one class of product that can claim conformance to this specification: a <dfn>user agent</dfn>.</p>
   </section>
+  
+  <section>
+    <h2>IANA Considerations</h2>
+    <p>The link relation types below have been registered by IANA per Section 6.2.1 of [[!RFC5988]]:</p>
+    
+    <section>
+      <h3>dns-prefetch Link Relation Type</h3>
+      <dl>
+        <dt>Relation Name:</dt>
+        <dd>dns-prefetch</dd>
+        <dt>Description:</dt>
+        <dd>The dns-prefetch link relation type is used to indicate an origin that will be used to fetch required resources, and that the user agent SHOULD resolve as early as possible.</dd>
```

Comment:
On 2016-01-25 14:44 , Ilya Grigorik wrote:
>> +        <dd>The dns-prefetch link relation type is used to indicate an origin that will be used to fetch required resources, and that the user agent SHOULD resolve as early as possible.</dd>
> No, that's not pretty either.

yes, that would be a rather ugly hack.

> Another idea: write own JS to pull out definitions from up higher and
> inject them here.

not sure it's worth the effort to write custom scripts to make up for 
respec's lack of macros. maybe we can ask the respec guys? sounds like a 
rather general and usual feature to me.

> Or, keep it simple and just "Please see |<link>|",
> where link is a reference to the definition. Surely they can just copy
> the right bits when they do the registration? =/

sure, but that's basically what i am doing here. copying the description 
so that i can link to the IANA section when sending the registration 
request to the list.

and honestly, i think you are overthinking this one. let's just copy 
manually for now, send the request, and get the registrations into the 
registry. and if things do get out of sync later on, probably not all 
that completely and it won't matter that much. i'll promise to keep an 
eye on it and will try to see that any changes do get reflected.